zhmccli/_cmd_imageprofile.py

```diff
@@ -0,0 +1,1918 @@
+# Copyright 2023 IBM Corp. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Commands for image activation profiles for CPCs in classic mode.
+"""
+
+from __future__ import absolute_import
+
+import copy
+import click
+from click_option_group import optgroup
+
+import zhmcclient
+from .zhmccli import cli
+from ._helper import print_properties, print_resources, abort_if_false, \
+    options_to_properties, original_options, COMMAND_OPTIONS_METAVAR, \
+    click_exception, add_options, LIST_OPTIONS, str2float, str2int
+from ._cmd_cpc import find_cpc
+from ._cmd_certificates import find_certificate
+
+
+# Defaults for image activation profile creation
+# DEFAULT_IFL_PROCESSORS = 1
```

Comment:
are those "your" defaults, or what the API endpoint will "fill in" automatically?

Reply:
These should not be there, and also not in the load profile module. They were copied from another file with the intention to define defaults, but it turns out the defaulting for activation profiles works by using a default profile.

I have removed these comments, and have added a paragraph explaining the defaulting to the help text of the create commands (it was already in the description of the `--copy-name` option, but I thought it should be mentioned more prominently.